packages/camera/camera/android/src/test/java/io/flutter/plugins/camera/CameraCaptureCallbackStatesTest.java

```diff
@@ -0,0 +1,373 @@
+package io.flutter.plugins.camera;
```

Comment:
Missing license header:
```suggestion
// Copyright 2013 The Flutter Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

package io.flutter.plugins.camera;
```